CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ set(CMAKE_MODULE_PATH
     ${CMAKE_MODULE_PATH}
     "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
-find_package(Boost REQUIRED COMPONENTS program_options system thread)
+find_package(Boost REQUIRED COMPONENTS program_options system thread regex)
```

Comment:
Well, the build works as is without regex on Linux. And this is the platform I care about.